mod/aalborg_theme/views/default/css/elements/misc.php

```diff
@@ -62,3 +62,22 @@
 	display: none;
 	padding: 0;
 }
+
+/* ***************************************
+	GLOBAL AJAX SPINNER
+*************************************** */
+
+.elgg-wait-cursor,
+.elgg-wait-cursor * {
+	cursor:wait !important
```

Comment:
Why does this need to use ```!important```? Usually the need to use it means that there's something wrong with the CSS elsewhere.